facts.d/example_control_repo.yaml

```diff
@@ -0,0 +1,5 @@
+# This is an example of how to use the control_repo_path custom fact
+# for deploying your own control repo instead of the default.
+#
+#control_repo_path: 'https://github.com/thespain/control-repo.git'
```

Comment:
You may also want to name this file `foo.yaml` to match the fake fact and note in here that they will need to create `control_repo_path.yaml` with the associated real setting in it. In the `.gitignore` you can add this to make it work well:

```
facts.d/*
!facts.d/foo.yaml
```